docs/scenarios/parametric_contributing.md

```diff
@@ -0,0 +1,60 @@
+# Contributing to Parametric System-tests
+
+Note: a more in-depth overview of parametric system-tests can be found in [parametric.md](parametric.md).
+
+**MUST:** Acquaint yourself with [this section](parametric.md#architecture-how-system-tests-work) for reference so you understand/can track what system-tests are actually doing.
+
+## Use cases
+
+Let's figure out if your feature is a good candidate to be tested with parametric system-tests. Parametric system-tests are great for assuring uniform behavior between tracers e.g. [environment variable configuration effects on api methods, sampling, propagation, configuration, telemetry](/tests/parametric).
+
+The parametric tests rely on the hitting of [http endpoints](/tests/parametric) that run tracer methods to produce and modify spans (manual instrumentation). If you'd like to test behavior across automatic instrumentations of tracers then you should assess if weblog system-tests may be a better fit.
+
+Parametric system-tests are horrible for testing internal or niche tracer behavior. Tests for those should exist on the tracer repos since they're only applicable for that specific tracer.
+
+## Getting setup
+
+Usually the one writing the system-tests is writing for a new feature, potentially one that hasn't been completed across all tracers yet. Therefore they'll want to focus on writing and getting the tests to pass for their tracer implementation first.
+
+To begin we need to point system-tests towards a tracer that has the feature implemented (published or on a branch).
+Follow [Binaries Documentation](../execute/binaries.md) for your particular tracer language to set this up.
+
+[Try running the tests for your tracer language](parametric.md#running-the-tests) and make sure some pass (no need to run the whole suite, you can stop the tests from running with `ctrl+c`). If you have an issue, checkout the [debugging section](parametric.md#debugging) to troubleshoot.
+
+## Writing the tests
+
+Now that we're all setup with a working test suite and a tracer with the implemented feature, we can begin writing the new tests.
+
+First take a look at the [currently existing tests](/tests/parametric) and see if what you're trying to test is similar and can use the same methods/endpoints (in many cases this is true).
+
+For all of the exact methods already implemented you can take a look at `class APMLibrary` in the [_library_client.py](/utils/parametric/_library_client.py). If you're wondering exactly what the methods do, you can take at look at the respective endpoints they're calling in that same file in `class APMLibraryClient`.
+
+The endpoints (where the actual tracer code runs) are defined in the Http Server implementations per tracer [listed here](parametric.md#http-server-implementations). Click on the one for your language to take a look at the endpoints. In some cases you may need to just slightly modify an endpoint rather than add a new one.
```

Comment:
I feel it would be helpful to define "Parametric servers" as the servers that serve requests (via endpoints) from APMLibraryClient. They perform tracing activity and send relevant data about the resulting trace(s) back to the client.

Reply:
I feel like we should rely on the documentation in parametric.md for this understanding, which is why I link to it extensively and ask users to read the section of how system-tests work before starting in on this doc. I'll add a reminder to reference that section if confused.